WORKSPACE

```diff
@@ -0,0 +1,50 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+android_sdk_repository(
+    name = "androidsdk",
+    api_level = 28,
+    build_tools_version = "28.0.2",
+)
+
+#Add support for JVM rules: https://github.com/bazelbuild/rules_jvm_external
+RULES_JVM_EXTERNAL_TAG = "2.9"
+RULES_JVM_EXTERNAL_SHA = "e5b97a31a3e8feed91636f42e19b11c49487b85e5de2f387c999ea14d77c7f45"
+http_archive(
+    name = "rules_jvm_external",
+    strip_prefix = "rules_jvm_external-%s" % RULES_JVM_EXTERNAL_TAG,
+    sha256 = RULES_JVM_EXTERNAL_SHA,
+    url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
+)
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+# Add support for Kotlin: https://github.com/bazelbuild/rules_kotlin.
+RULES_KOTLIN_VERSION = "legacy-1.3.0-rc4"
+RULES_KOTLIN_SHA = "fe32ced5273bcc2f9e41cea65a28a9184a77f3bc30fea8a5c47b3d3bfc801dff"
+http_archive(
+    name = "io_bazel_rules_kotlin",
```

Comment:
Should we import the temporary version we need for data-binding?

Reply:
I was thinking of waiting to change this to the data-binding version until I begin working on the /app module - but I can change it now if you think it is best to do so!

Reply:
Quick update: I tried to use git_repository() to add the version of rules_kotlin that contains the PR you wrote and it is giving me an error about circular dependencies within the WORKSPACE file. It looks like git_repository() might be the problem. See https://github.com/bazelbuild/bazel/issues/10062.

Reply:
Yes! Just pushed the change!